doc/assets.md

```diff
@@ -10,41 +10,37 @@ For assets, a special **`ezdesign` asset package** is available.
 <img src="{{ asset("images/foo.png", "ezdesign") }}" alt="foo"/>
 ```
 
-Using `ezdesign` package will resolve current design with theme fallback.
+Using the `ezdesign` package will resolve current design with theme fallback.
 
 By convention, an asset theme directory can be located in:
-* `<bundle_directory>/Resources/public/themes/`
-* `web/assets/themes/`
+- `<bundle_directory>/Resources/public/themes/`
```

Comment:
With the perspective of moving all doc to GitHub, we are trying to formulate md standards to use for doc which, while not binding for contributors, would make maintaining and editing the doc easier. This was also the reason for making the changes to line lengths.
However, if this is not accepted, I'll roll back the changes.